### DISCUSSION TITLE
Know how ppc is constructed (used for every further steps in poweflow

### DISCUSSION THREAD

**Discussion by Arjunayodya** [e2nIEE/pandapower#1570] [Q&A]
I am curious to know how the PPC dictionary is constructed, which is used for every further calculation. So I would like to the theory (Python implementation). I would be grateful if someone share how to access / how it is constructed

**Comment by SteffenMeinecke** [comment]
ppc (pypower case) dictionaries are used internal data format within pandapower. It is based on the [pypower package](https://pypi.org/project/PYPOWER/) which is a port of matlab's [matpower](https://matpower.org/). The latter is well documented including a good manual.
